content/download/windows.en.md

```diff
@@ -34,28 +34,21 @@ Standalone installer: install GRASS GIS with the required support packages.
 #### <a name="GRASS-GIS-old-stable"></a> GRASS GIS 7.6.1 (legacy)
 
 <div class="alert rounded-0 alert-warning">
-<i class="fa fa-info-circle"></i> <u>Old stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures76">GRASS GIS 7.6.1 new features</a> and  <a href="https://trac.osgeo.org/grass/wiki/Release/7.6.1-News">GRASS GIS 7.6.1 announcement</a> for more information.
+<i class="fa fa-info-circle"></i> <u>Old stable release</u>, see <a href="https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures76">GRASS GIS 7.6 new features</a> and  <a href="https://trac.osgeo.org/grass/wiki/Release/7.6.1-News">GRASS GIS 7.6.1 announcement</a> for more information.
 </div>
 
-
 *  [<i class="fa fa-download"></i> Download 64bit](/grass76/binary/mswindows/native/x86_64/WinGRASS-7.6.1-1-Setup-x86_64.exe)
 *  [<i class="fa fa-download"></i> Download 32bit](/grass76/binary/mswindows/native/x86/WinGRASS-7.6.1-1-Setup-x86.exe)
 
 <hr>
 
-#### <a name="GRASS-GIS-devel"></a> GRASS GIS 8.0 (preview)
+#### <a name="GRASS-GIS-devel"></a> GRASS GIS 8.0
 
 <div class="alert rounded-0 alert-info">
-<i class="fa fa-info-circle"></i> Active <u>development</u> and <u>experimental</u> <b>GRASS GIS</b> version.
+<i class="fa fa-info-circle"></i> Active <u>development</u> and <u>new stable</u> <b>GRASS GIS</b> version.
 </div>
 
-
-*  [<i class="fa fa-download"></i> Download 64bit](https://wingrass.fsv.cvut.cz/grass80/x86_64)
-*  [<i class="fa fa-download"></i> Download 32bit](https://wingrass.fsv.cvut.cz/grass80/x86)
```

Comment:
Only this one is broken/non existing as we agreed to drop support for 32bit, but the 64bit link works. Shall we remove that one too?

Reply:
Also, as it says "new stable" for 7.8, I'd still probably keep the experimental, until we really have 8 out and it becomes the stable version. Otherwise, we would be out of sync with linux and mac pages... What do you think?

Reply:
> Only this one is broken/non existing as we agreed to drop support for 32bit, but the 64bit link works. Shall we remove that one too?

Honestly, I find a link to an outdated version from Nov 2021 with wrong file name confusing, that's why I remove it.

Reply:
Maybe we can now add the link to RC1 once it's fixed by @landam, just to not leave the entry empty.

Reply:
Fine for me.